rollup.config.js

```diff
@@ -137,4 +139,7 @@ const config = [
 	}
 ];
 
+// For development we are using `umd`s
+if (PROD) config.unshift(esmCjsConfig);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add curly braces around the nested statement(s) in this "if" block. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=javascript%3ACurlyBraces)